src/main/java/com/jfrog/ide/idea/scan/ScanManagersFactory.java

```diff
@@ -84,13 +93,36 @@ public void startScan() {
                     logError(Logger.getInstance(), "", e, true);
                 }
             }
+        } catch (JFrogInactiveEnvironmentException e) {
+            handleJfrogInactiveEnvironment(e.getRedirectUrl());
```

Comment:
Shouldn't we do this also in the test connection button?
Same question for CI mode.

Reply:
Added this to the test connection:
<img width="720" alt="Screenshot 2023-02-19 at 18 15 56" src="https://user-images.githubusercontent.com/24526180/219961845-3b54d0dd-2d32-458a-8e03-8155e2cf95f5.png">
Regarding the CI view, I don't think it is necessary - an Error will be displayed in the logs.